qt doesn't expose the x11 headers to the world, so no need to make it required.

--- a/cmake/modules/FindQt4.cmake
+++ b/cmake/modules/FindQt4.cmake
@@ -1457,10 +1457,9 @@
   IF(UNIX)
     # on OS X X11 may not be required
     IF (Q_WS_X11)
-      FIND_PACKAGE(X11 REQUIRED)
+      FIND_PACKAGE(X11)
     ENDIF (Q_WS_X11)
     FIND_PACKAGE(Threads)
-    SET(QT_QTCORE_LIBRARY ${QT_QTCORE_LIBRARY} ${CMAKE_THREAD_LIBS_INIT})
   ENDIF(UNIX)
 
 
